htdocs/admin/dict.php

```diff
@@ -1260,6 +1261,15 @@
 $title = $langs->trans("DictionarySetup");
 
 llxHeader('', $title, '', '', 0, 0, '', '', '', 'mod-admin page-dict');
+$object = new modSocialNetworks($db);
+
+if (preg_match('/[?&]id=([0-9]+)/', $object->config_page_url[0], $matches)) {
+	$id = $matches[1];
+}
+
+$head = socialnetwork_prepare_head($id);
+
+print dol_get_fiche_head($head, 'dict', $langs->trans('MenuDict'), -1, 'user');
```

Comment:
If you add a print 
print dol_get_fiche_head(),
you should also add a 
print dol_get_fiche_end()